superset/commands/export/assets.py

```diff
@@ -50,15 +51,17 @@ def run(self) -> Iterator[tuple[str, Callable[[], str]]]:
             ExportDatabasesCommand,
             ExportDatasetsCommand,
             ExportChartsCommand,
+            ExportTagsCommand,
             ExportDashboardsCommand,
             ExportSavedQueriesCommand,
         ]
         for command in commands:
-            ids = [model.id for model in command.dao.find_all()]
-            for file_name, file_content in command(ids, export_related=False).run():
-                if file_name not in seen:
-                    yield file_name, file_content
-                    seen.add(file_name)
+            if hasattr(command, "dao"):

```

Comment:
I'm confused by this — does `ExportTagsCommand` have a DAO? If it has, why the `hasattr` check; if it doesn't, why include it in `commands`?

Reply:
My first thought was since it is export I "think" I should add it in the Assets commands, but no ExportTagsCommand  does not use dao at all. We have removed it in the `commands`. Thanks!!